internal/transport/grpcframer/framer.go

```diff
@@ -0,0 +1,267 @@
+/*
+ *
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package grpcframer defines the interface for implementing an HTTP/2 framer,
+// its required types and an implementation of said framer.
```

Comment:
Nit: this comment is pretty verbose. "defines an interface and implementation of an HTTP/2 framer" or something along those lines.

Reply:
You probably can omit a mention to the `required types`, but not a big deal for me :)